apps/console/src/features/users/components/guests/pages/invite-parent-org-user.tsx

```diff
@@ -60,23 +42,8 @@ interface InviteParentOrgUserFormErrorsInterface {
  * Props interface of {@link InviteParentOrgUser}
  */
 interface InviteParentOrgUserPropsInterface extends IdentifiableComponentInterface {
-    /**
-     * Callback method for closing the wizard.
-     */
-    closeWizard: () => void;
-    /**
-     * Callback method for setting the isSubmitting state.
-     */
-    setIsSubmitting: (isSubmitting: boolean) => void;
-    /**
-     * The Callback method for when the user invite is successful.
-     */
-    onUserInviteSuccess: () => void;
-    /**
-     * The callback method for setting the alert.
-     * @param alert - The alert object.
-     */
-    setAlert: (alert: WizardAlert) => void;
+    onSubmit?: (values: any) => void;
```

Comment:
I think we can use Map<string, FormValue> here for the `values` type, instead of any

Reply:
Changed